dashboard/src/components/AppView/CustomAppView/CustomAppView.test.tsx

```diff
@@ -59,6 +59,20 @@ const defaultProps = {
   appDetails: {} as AvailablePackageDetail,
 };
 
+// Ensure remote-component doesn't trigger external requests during this test.
+const xhrMock: Partial<XMLHttpRequest> = {
+  open: jest.fn(),
+  send: jest.fn(),
+  setRequestHeader: jest.fn(),
+  readyState: 4,
+  status: 200,
+  response: "Hello World!",
+};
+
+beforeAll((): void => {
+  jest.spyOn(window, "XMLHttpRequest").mockImplementation(() => xhrMock as XMLHttpRequest);
+});
```

Comment:
Or this (which landed). Guessing you merged master into this PR but hadn't done so on the base branch that this PR targets?